src/IndGenTR.jl

```diff
@@ -0,0 +1,36 @@
+# Indicator of a generalized trust region.
+
+export IndGenTR
+
+"""
+Indicator function of a trust region defined by two n-dimensionnal vectors l and u.
+Check component wise if the elements are between the components of l and u.
+"""
+
+struct IndGenTR
+    l::AbstractVector{Real}
```

Comment:
We use 2 spaces for indentation. Could you please update your VSCode configuration?